packages/mgt/src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -695,10 +724,23 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
           }
         }
 
-        if ((this.type === PersonType.group || this.type === PersonType.any) && people.length < this.showMax) {
-          people = [];
+        if (this.type === PersonType.group || (this.type === PersonType.any && people.length < this.showMax)) {
```

Comment:
I'd change this to check for the length of the people array regardless whether it's `group` or `any`

```suggestion
        if ((this.type === PersonType.group || this.type === PersonType.any) && people.length < this.showMax) {
```

Reply:
It's actually Prettier that changed the logic here. I agree it should always be checked!